lib/releaselib.sh

```diff
@@ -127,7 +127,7 @@ release::set_build_version () {
   local -a all_jobs=($($GHCURL \
    $K8S_GITHUB_RAW_ORG/test-infra/master/testgrid/config/config.yaml \
    2>/dev/null |\
-   $yq -r '.[] | .[] | select (.name?=="sig-'$branch'-blocking") |.dashboard_tab[].test_group_name' 2>/dev/null))
+   $yq -r '.[] | .[] | select (.name?=="sig-release-'$branch'-blocking") |.dashboard_tab[].test_group_name' 2>/dev/null))
```

Comment:
See above 'deal with inconsistent naming in config.yaml'.  Is this related?  Did something change there?